pkg/envtest/server.go

```diff
@@ -334,12 +334,13 @@ func (te *Environment) startControlPlane() error {
 }
 
 func (te *Environment) waitForDefaultNamespace(config *rest.Config) error {
+	fmt.Println("waiting...")
```

Comment:
If we want to add a log here. Let's please also write what we are waiting for and log the same way as in other places (e.g. l.299: `log.V(1).Info("installing webhooks")`)

Reply:
Shoot. Yep, you're totally right. I don't even think that we want to log here. This was just an ad-how change that I forgot to remove before pushing

Reply:
Updated.